irods/test/data_obj_test.py

```diff
@@ -654,6 +656,61 @@ def test_multiple_reads(self):
             with obj.open('r') as f:
                 self.assertEqual(f.read().decode(), obj.path)
 
+    def _skip_unless_connected_to_this_computer_by_other_than_localhost_synonym(self):
+        if self.sess.host not in ({socket.gethostname()} - {'localhost', '127.0.0.1'}):
```

Comment:
well, all of 127/8 if you want to be thorough

Reply:
Oh.  128.* too?